projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -997,7 +997,10 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
     /** @hidden @internal */
     public ngOnDestroy(): void {
         if (this._componentID) {
-            this._overlayService.hide(this._componentID);
+            this._overlayService.detach(this._componentID);
+        }
+        if (this._statusChanges$) {
```

Comment:
This duplicate seems like a leftover

Reply:
`hide` is deleted and `detach` is added. I do not see a duplicate or I am missing something?

Reply:
find it finally
leftover removed